ui/pages/confirmation/templates/create-snap-account.js

```diff
@@ -0,0 +1,26 @@
+function getValues(pendingApproval, t, actions, _history) {
+  const { origin: snapId, snapName } = pendingApproval;
+
+  return {
+    content: [
+      {
+        element: 'CreateSnapAccount',
+        key: 'create-snap-account',
+        props: {
+          snapId,
+          snapName,
+        },
+      },
+    ],
+    cancelText: t('cancel'),
+    submitText: t('create'),
+    onSubmit: () => actions.resolvePendingApproval(pendingApproval.id, true),
```

Comment:
Did we not need the input support to pass through the account name?

Reply:
We are blocked on using the input meaningfully by [this PR](https://github.com/MetaMask/metamask-extension/pull/20344) so I removed it for now.